secant/SecantApp.swift

```diff
@@ -9,11 +9,12 @@ import SwiftUI
 
 @main
 struct SecantApp: App {
-    @StateObject var appRouter = AppRouter(services: MockServices())
-    
+    var homeStore: HomeStore = .demo
     var body: some Scene {
         WindowGroup {
-            appRouter.rootView()
+            NavigationView {
+                HomeView(store: homeStore)
```

Comment:
Beware of the initial screen. I need to work with @geffenz to see how we will handle the app not being able to initialize because of critical errors like db migrations, rust backend errors that probably will require the user getting out the recovery phrase and actually nuking the wallet or in a better situation just trigger a rescan.

Reply:
Also there's a caveat with the keychain not being available at the very first moment of the app's initialization. In the previous app we had to implement a fake screen that actually gave iOS time to decrypt the keychain elements needed to verify the state of the wallet (existing credentials or empty) 

This is not blocking, but I wanted to bring it up since it's not on the designs

Reply:
Ah yeh really good point! We will definitely have to explicitely model this. That would be a pretty good thing to move on to after this to nail down the states / logic, taking what you've done previously 👍.